### ISSUE TITLE
SetWeaponsNoAutoReload

### ISSUE THREAD

**Issue by kyro95** [altmp/altv-issues#1948]
### Description of the problem

There's no way to prevent a player from not automatically reloading when magazine is empty

### Desired solution for the problem

Implementation of a flag on client side similiar to this;

https://docs.fivem.net/natives/?_0x311150E5

### Version

15.0-dev505 (dev)

**Comment by FabianTerhorst** [comment]
alt.setConfigFlag("DISABLE_AUTO_WEAPON_SWAP", true)

**Comment by FabianTerhorst** [comment]
Yes you are right we only have a weapon swap disabling, but no reload prevention atm.

**Comment by Yiin** [comment]
@FabianTerhorst `alt.setConfigFlag("DISABLE_AUTO_WEAPON_SWAP", true)` doesn't seem to work on latest dev

https://github.com/altmp/altv-issues/assets/9081457/565d20db-c640-4b3e-b198-0df146a22538
